docs/v5/configuration/htcondor-routes.md

```diff
@@ -113,6 +113,38 @@ To do this, replace the above `SET Requirements` or `set_Requirements` lines wit
     set_Requirements = original_requirements && ...;
     ```
 
+
+### Setting the accounting group according to the the credential that submitted the job ###
```

Comment:
```suggestion
### Setting the accounting group based on the credential of the submitted job ###
```

Or something along those lines. The credential didn't submit the job but it did come along with the submitted job.